### PR TITLE
Fix merging of existing cases broken in recent merges

### PR DESCRIPTION
**Pull request by kswenson**

### CODE REVIEW

File: src/lib/codap-helper.ts [hunk 1]
```diff
@@ -315,16 +331,18 @@ export class CodapHelper {
...
-          return !sharedAttributes.some(attrB => attrA.name === attrB.name);
-        });
-
-        changeCommands.push(...staleAttributes.map(attr => ({
-          action: "delete",
-          resource: collectionResource(dataContext.name, attr.collection, `attribute[${attr.name}]`)
-        })));
-      }
+      // Disabled for now, as we still have echo effects that sometimes
+      // cause attributes to be deleted incorrectly on initial join.
```
Comment (kswenson) [inline src/lib/codap-helper.ts]: Encountered more echo/rebound effects resulting in deleting of merged attributes. Disabling deletion of attributes on sync for the moment as correct merging is more important than deleting attributes on sync right now.

**Comment by kswenson** [comment]
Sorry, I should have specified that. It's required to handle joining a share for the first time with an existing table with existing items.

**Comment by sfentress** [comment]
But that previously worked with the existing code, right?

**Comment by kswenson** [comment]
Not that I'm aware of. Neither of the other paths affect existing items that don't yet have the collaborator key.

**Comment by sfentress** [comment]
Ah, right, it worked before when we were added the new username to `caseByIndex[0]`.

**Comment by kswenson** [comment]
Which is why we're doing it again. 😉

**Pull request merged**